src/objecttypes/conf/base.py

```diff
@@ -25,6 +25,34 @@
     "objecttypes.utils",
 ]
 
+INSTALLED_APPS.pop(
+    next(
+        index
+        for index, app in enumerate(INSTALLED_APPS)
+        if app == "two_factor.plugins.webauthn"
+    )
+)
+
+TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "DIRS": [Path(DJANGO_PROJECT_DIR) / "templates"],
+        "APP_DIRS": False,  # conflicts with explicity specifying the loaders
+        "OPTIONS": {
+            "context_processors": [
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
+                "django.contrib.auth.context_processors.auth",
+                "django.contrib.messages.context_processors.messages",
+                "django.contrib.messages.context_processors.messages",
+                "open_api_framework.context_processors.project",
```

Comment:
I think if you add this line to the `TEMPLATES` in `open_api_framework/conf/base.py` we can avoid this duplicated code